ui/lib/core/addon/components/kv-object-editor.hbs

```diff
@@ -53,21 +53,19 @@
             />
           {{/if}}
         </div>
-        <div class="column is-narrow">
+        <div class="column is-1">
```

Comment:
`is-1` keeps both buttons the same width 
<img width="1141" alt="Screenshot 2023-11-14 at 3 29 56 PM" 
src="https://github.com/hashicorp/vault/assets/68122737/71d01c3c-a7d0-4da3-8dc1-29dc7c86289a">